app/utils/general.js

```diff
@@ -52,7 +53,7 @@ export function emptyFunction() { // eslint-disable-line no-empty-function
 
 export function hapticFeedback(method = 'impactLight') {
     ReactNativeHapticFeedback.trigger(method, {
-        enableVibrateFallback: true,
+        enableVibrateFallback: DeviceTypes.ENABLE_HAPTIC_FEEDBACK,
```

Comment:
Not sure why the changes in the constants, I thought that passing false here would be enough.

Reply:
If set to false this would disable it all together, we wanted to only disable it for devices that don't support hapticFeedback.

Reply:
Ok, so appears I 100% read the repo wrong on this! I'll have this fixed this evening!

Reply:
Ok fixed! Thanks guys!